quinn-proto/src/lib.rs

```diff
@@ -54,6 +54,9 @@ pub const VERSION: u32 = 0xff00_0011;
 /// TLS ALPN value for HTTP over QUIC
 pub const ALPN_QUIC_HTTP: &[u8] = b"hq-17";
 
+/// TLS ALPN value for H3
+pub const ALPN_QUIC_H3: &[u8] = b"h3-17";
```

Comment:
```suggestion
pub const ALPN_QUIC_H3: &[u8] = b"h3-18";
```
As of #225 (whoops, looks like I forgot this above too, feel free to get them both)

Reply:
fixed in 8569932